### ISSUE TITLE
Fedora Repo returns wrong Content-Length

### ISSUE THREAD

**Issue by CaptainTyphoo** [iridium-browser/tracker#140]
I added the Fedora repo, but when I try installing iridium, I get the following error message:

```
Pakete werden heruntergeladen:
[MIRROR] iridium-browser-57.0-2.fc25.x86_64.rpm: Interrupted by header callback: Server reports Content-Length: 66502656 but expected size is: 180365026
[FAILED] iridium-browser-57.0-2.fc25.x86_64.rpm: No more mirrors to try - All mirrors were already tried without success
(2/2): iridium-browser-ffmpegsumo 99% [===============================-] 162 GB/s | 172 MB     00:00 ETA
Die heruntergeladenen Pakete wurden bis zur nächsten erfolgreichen Transaktion im Zwischenspeicher abgelegt.
Sie können zwischengespeicherte Pakete mit dem Befehl »dnf clean packages« entfernen.
Fehler: Error downloading packages:
  Cannot download iridium-browser-57.0-2.fc25.x86_64.rpm: All mirrors were tried

```

**Comment by souldanger** [comment]
Hi Typhoo!
we are experiencing problems with these files on our servers.
Please stand-by - this will be solved shortly!

Thanks you for your patience!

**Comment by z1mvader** [comment]
Hey guys! I'm very excited to try Iridium but I'm having the same problem

**Comment by ngothan** [comment]
it's fixed in latest 58.0
